setup.py

```diff
@@ -2,10 +2,10 @@
 
 from setuptools import find_packages, setup
 
-with open("base.txt") as f:
```

Comment:
Hmm... we don't keep the `requirements` folder when copying to the Docker image, that's why it was changed here

Reply:
In Dockerfile, we use `pip install -r ...`, but e.g. `tox` is installing via `setup.py` and it was broken. This fixes it, or is it a bad fix?

Reply:
We also do `pip install -e .`, but from what I can see in the CI it works fine. So let's merge it